setup/pkg/fieldlabs/environment_create.go

```diff
@@ -608,7 +608,7 @@ func (e *EnvironmentManager) inviteUsers(envs []Environment, policies map[string
 		}
 		inviteBody := map[string]string{
 			"email":     env.Email,
-			"policy_id": policies[e.getAppName(env)],
+			"policy_id": policies[env.App.ID],
```

Comment:
I'd change this back to `e.getAppName(env)` in order to send the invite and match the policy name.